scalastyle-config.xml

```diff
@@ -0,0 +1,161 @@
+<!--
+  ~ Copyright 2018 ABSA Group Limited
```

Comment:
Probably taken from some other project (which one? 😉 )
The license year should be the year of project establishment. (2021 this one, I believe)

Reply:
From Atum

Reply:
changed